### ISSUE TITLE
Bot doesn't come in the audio channel

### ISSUE THREAD

**Issue by etn406** [codetheweb/muse#862]
**What's happening?**
I used Muse a year ago (in a Docker container), it worked, and after some time the container crashed, I stopped it and it wasn't used until now. I deleted it, deleted the old data volume, kicked it from the server, and recreated a clean, up-to-date Docker container from `codetheweb/muse`.

I invited the bot to my Discord server with the link in the Muse logs, and two unusual things I noticed:
* the bot doesn't show up in Discord's right sidebar with the rest of the server members
* the bot doesn't send me any private message

And then, the problem:
* The bot doesn't come in the audio channel to play music. It is correctly responding to text commands. If I query and play a song with `/play` it will correctly show the results and respond, but doesn't connect to the audio channel. When `/now-playing` is used, the song looks like it is currently playing.

**Logs**

[Muse logs from the starting of the server and with DEBUG=*](https://github.com/codetheweb/muse/files/10429809/muse.csv)

**Screenshots**

Text commands look normal but the bot isn't present in the audio channel.

<img width="781" alt="Capture d’écran 2023-01-16 à 23 10 20" src="https://user-images.githubusercontent.com/1438257/212773572-ee431c1d-607a-4956-bc35-acabb2c2ee85.png">

**Runtime**
I'm running Muse:
- [ ] Directly from the cloned repository
- [x] Inside a Docker container
- [ ] Something else (please elaborate)

**Versions**
- Muse: 2.1.7
- Docker (if applicable): 20.10.3, build 55f0773
- OS: DSM 7.1.1-42962 Update 2 (Linux for DiskStation NAS)

**Comment by codetheweb** [comment]
Hmm, it sounds like the permissions are messed up but it's weird that nothing is showing up in the logs.

Could you try creating a fresh Discord guild and inviting the bot there?

**Comment by etn406** [comment]
It works on a fresh Discord guild:
- the bot joins the server and is visible in the members sidebar,
- the bot sends me a message,
- the bot plays music in the audio channel when asked.

I will compare guilds permissions between this one and the previous one, to try to understand why it doesn't work the same. If you have a better idea, I'm all ears 😄

**Comment by etn406** [comment]
Ok, so this are the bot's permissions on the fresh guild:
<img width="668" alt="Capture d’écran 2023-01-18 à 13 18 37" src="https://user-images.githubusercontent.com/1438257/213169726-8f36453b-52be-476b-96fe-ff6163685420.png">

And this is on my main guild:
<img width="668" alt="Capture d’écran 2023-01-18 à 13 19 49" src="https://user-images.githubusercontent.com/1438257/213169843-8c525650-8db1-4ef5-90db-b38859789acf.png">

I'll now try to give the _MuseApp_ role the same permissions on my main guild, and then kick and reinvite the bot.

**Comment by etn406** [comment]
Ok, I manage to have the bot working on my main guild. My guess is that the issue was because of the role _@everyone_ having no permissions at all, and the _MuseApp_ inherited from it everytime I added it to the server.

I fixed it by giving back to the roles _@everyone_ and _MuseApp_ all the default permissions, then removing the bot's integration and reinviting it to the server.

It didn't send me any private message when joining, but I now can see it in the member list, and it can play music in the audio channels 🥳

Thanks for putting me on the right track! 😄
